cirq_superstaq/ops/qubit_gates.py

```diff
@@ -246,71 +254,111 @@ def _num_qubits_(self) -> int:
+            return args.format("acecr_rx({:half_turns}) {},{};\n", exponent, *qubits)
+        else:
+            exponent = self.theta / np.pi
+            return args.format("acecr_rzx({:half_turns}) {},{};\n", exponent, *qubits)
 
     def _is_parameterized_(self) -> bool:
-        return cirq.is_parameterized(self.sandwich_rx_rads)
+        return cirq.is_parameterized(self.sandwich_rx_rads) or cirq.is_parameterized(self.theta)
 
     def _parameter_names_(self) -> AbstractSet[str]:
-        return cirq.parameter_names(self.sandwich_rx_rads)
+        return cirq.parameter_names(self.sandwich_rx_rads) | cirq.parameter_names(self.theta)
 
     def _resolve_parameters_(self, resolver: cirq.ParamResolver, recursive: bool) -> "AceCR":
         return AceCR(
-            self.polarity, cirq.resolve_parameters(self.sandwich_rx_rads, resolver, recursive)
+            theta=cirq.resolve_parameters(self.theta, resolver, recursive),
+            sandwich_rx_rads=cirq.resolve_parameters(self.sandwich_rx_rads, resolver, recursive),
         )
 
-    def _value_equality_values_(self) -> Tuple[str, cirq.TParamVal]:
+    def _value_equality_values_(self) -> Tuple[cirq.TParamVal, cirq.TParamVal]:
+        if cirq.is_parameterized(self.theta):
+            theta = self.theta
+        else:
+            theta = self.theta % (2 * np.pi)
+
         if cirq.is_parameterized(self.sandwich_rx_rads):
-            return self.polarity, self.sandwich_rx_rads
+            sandwich_rx_rads = self.sandwich_rx_rads
+        else:
+            sandwich_rx_rads = self.sandwich_rx_rads % (4 * np.pi)
 
-        return self.polarity, self.sandwich_rx_rads % (4 * np.pi)
+        return theta, sandwich_rx_rads
 
-    def _value_equality_approximate_values_(self) -> Tuple[str, cirq.PeriodicValue]:
-        return self.polarity, cirq.PeriodicValue(self.sandwich_rx_rads, 4 * np.pi)
+    def _value_equality_approximate_values_(self) -> Tuple[cirq.PeriodicValue, cirq.PeriodicValue]:
+        return (
+            cirq.PeriodicValue(self.theta, 2 * np.pi),
+            cirq.PeriodicValue(self.sandwich_rx_rads, 4 * np.pi),
+        )
 
     def _equal_up_to_global_phase_(self, other: Any, atol: float) -> Optional[bool]:
         if not isinstance(other, AceCR):
             return NotImplemented
 
-        if other.polarity != self.polarity:
-            return False
-
-        return approx_eq_mod(self.sandwich_rx_rads, other.sandwich_rx_rads, 2 * np.pi, atol=atol)
+        return approx_eq_mod(self.theta, other.theta, 2 * np.pi, atol=atol) and approx_eq_mod(
+            self.sandwich_rx_rads, other.sandwich_rx_rads, 2 * np.pi, atol=atol
+        )
 
     def __repr__(self) -> str:
-        if not self.sandwich_rx_rads:
-            return f"css.AceCR({self.polarity!r})"
-        return f"css.AceCR({self.polarity!r}, {self.sandwich_rx_rads!r})"
+        if not self.sandwich_rx_rads and self.theta == np.pi / 2:
+            return "css.AceCR"
```

Comment:
this should be
```suggestion
            return "css.AceCR()"
```
so that `eval(repr(gate)) == gate`